RELEASE_NOTES

```diff
@@ -1,3 +1,10 @@
+========================================================================
+Amazon CloudWatch Agent 1.300049.0 (2024-10-30)
+========================================================================
+Enhancements:
+* [Metrics/Logs] Annotate Infra, statsD, collectD metrics and Fluent-bit application logs with entities
```

Comment:
probably remove fluent-bit..

we can optionally say  "[ApplicationSignals] Create a server to expose service names for instrumented applications on K8s"